minetest/minetest
# Fix gettext detection and locale building (broken since a368e7e)

This PR fixes cmake gettext detection and locale generation broken since a368e7e, also might solve #9771

- Fix enabling gettext broken since a368e7e by replacing wrong variable (GETTEXT_FOUND) with correct (GETTEXTLIB_FOUND) in `src/CMakeLists.txt`, to make sure variable name matches the new variable name provided in `cmake/Modules/FindGettextLib.cmake`
- Result fix is correct setting of USE_GETTEXT variable and correct locale file generation and building with gettext.

## To do

This PR is a Ready for Review.

## How to test

- (optional) Clean git source tree by running command `git clean -d -x -i` (You can exclude files by filtering listed files by pattern as prompted while using this command)
- Rerun cmake and observe its' output (should include line beginning with `GetText enabled; locales found`), and after running make minetest should be translated like it was before mentioned commit was merged.
